Content.Client/GameObjects/Components/Observer/AcceptCloningWindow.cs

```diff
@@ -34,7 +34,7 @@ public AcceptCloningWindow()
                                 {
                                     (AcceptButton = new Button
                                     {
-                                        Text = Loc.GetString("Yes"),
+                                        Text = Loc.GetString("generic-yes"),
```

Comment:
Ah, bit of a problem here, some languages have context dependent "yes" and "no"s!

As an example from the top of my head, Japanese yes and no don't function like english yes and no, in that they are used to agree with the question raised, as opposed to answering the question.

E.g. the question "Do you want to go home?" implies that you do, and so if you do "Yes" is the natural response.
An equivalent, negatively implicated question, "You don't want to go home?" if you didn't want to go home the natural english response is "No" as in "No I do not want to go home" (answering the question).
The natural response here in japanese is "Yes" as in "Yes, your assumption is correct, I do not want to go home".

The long and short of it is, be careful with generic "Yes" and generic "No!" (as in, please make these specific where possible, relate them to the question being asked.)